File: movies-app/src/app/models/movie.ts
export class Movie {

  public title: string;

  public released: string;

  public imdbRating: string;

  public metascore: string

  public tomatoMeter: string;

  public imdbVotes: string;

  public plot: string;

  public runtime: string;

  public director: string;

  public awards: string;

  constructor (title: string, released: string,
               imdbRating: string, metascore: string,
               tomatoMeter: string, imdbVotes: string,
               plot: string, runtime: string,
               director: string, awards: string) {
    this.title = title;

    this.released = released;

    this.imdbRating = imdbRating;

    this.metascore = metascore;

    this.tomatoMeter = tomatoMeter;

    this.imdbVotes = imdbVotes;

    this.plot = plot;

    this.runtime = runtime;

    this.director = director;

    this.awards = awards;
  }
}

File: Movies/src/Movies/Models/Pocos/Movie.cs
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Movies.Models.Pocos
{
    public class Movie
    {
        [Key]
        [Column(Order = 0)]
        [MaxLength(128)]
        public string Title { get; set; }

        [Key]
        [Column(Order = 1)]
        public string Released { get; set; }

        [Required]
        [MaxLength(4)]
        public string ImdbRating { get; set; }

        [Required]
        [MaxLength(4)]
        public string Metascore { get; set; }

        [Required]
        [MaxLength(4)]
        public string TomatoMeter { get; set; }

        [Required]
        [MaxLength(128)]
        public string ImdbVotes { get; set; }

        [Required]
        [MaxLength(512)]
        public string Plot { get; set; }

        [Required]
        [MaxLength(32)]
        public string Runtime { get; set; }

        [Required]
        [MaxLength(64)]
        public string Director { get; set; }

        [Required]
        [MaxLength(128)]
        public string Awards { get; set; }

        #region Relationships

        public ICollection<MovieGenre> MovieGenres { get; set; }

        #endregion
    }
}